{"uid":"88b756bf3b65ad31","name":"Get Template","fullName":"test_cases.test_template.TestTemplateManagement#test_get_template","historyId":"3e0654fc84f60fdcfbe40b395f79c630","time":{"start":1731823934299,"stop":1731823934425,"duration":126},"status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"autonomous_agent_api","time":{"start":1731823445286,"stop":1731823445286,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"admin_login_response","time":{"start":1731823447897,"stop":1731823448024,"duration":127},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"create_template_fixture","time":{"start":1731823448025,"stop":1731823448161,"duration":136},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"admin_login_cookie","time":{"start":1731823448025,"stop":1731823448025,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"afterStages":[{"name":"autonomous_agent_api::0","time":{"start":1731823934751,"stop":1731823934751,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"parentSuite","value":"Template Management"},{"name":"feature","value":"Get Template"},{"name":"suite","value":"test_template"},{"name":"subSuite","value":"TestTemplateManagement"},{"name":"host","value":"fv-az738-625"},{"name":"thread","value":"1985-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test_cases.test_template"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":8,"broken":15,"skipped":0,"passed":57,"unknown":0,"total":80},"items":[{"uid":"8dc0d2d4db174c6e","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/77//#testresult/8dc0d2d4db174c6e","status":"passed","time":{"start":1731476631077,"stop":1731476631261,"duration":184}},{"uid":"b9426a0fe0622959","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/77//#testresult/b9426a0fe0622959","status":"passed","time":{"start":1731475967230,"stop":1731475967352,"duration":122}},{"uid":"d2a22bc794195437","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/76//#testresult/d2a22bc794195437","status":"passed","time":{"start":1731474812596,"stop":1731474812774,"duration":178}},{"uid":"212487ad06808ded","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/75//#testresult/212487ad06808ded","status":"passed","time":{"start":1730875812370,"stop":1730875812495,"duration":125}},{"uid":"28b9940ddad5b287","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/74//#testresult/28b9940ddad5b287","status":"passed","time":{"start":1730865941361,"stop":1730865941480,"duration":119}},{"uid":"7b582e70ce11b5af","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/73//#testresult/7b582e70ce11b5af","status":"passed","time":{"start":1730186919901,"stop":1730186920085,"duration":184}},{"uid":"646cb2b8f755006a","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/72//#testresult/646cb2b8f755006a","status":"passed","time":{"start":1730186749832,"stop":1730186749952,"duration":120}},{"uid":"7d700bca7f59e78a","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/71//#testresult/7d700bca7f59e78a","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/auth/login (Caused by SSLError(SSLCertVerificationError(1, \"[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'api.agents.cardanoapi.io'. (_ssl.c:1000)\")))","time":{"start":1729748685847,"stop":1729748685847,"duration":0}},{"uid":"9f24ca6670d0fb3e","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/70//#testresult/9f24ca6670d0fb3e","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/auth/login (Caused by SSLError(SSLCertVerificationError(1, \"[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'api.agents.cardanoapi.io'. (_ssl.c:1000)\")))","time":{"start":1729680164243,"stop":1729680164243,"duration":0}},{"uid":"ba22627ebbd8f334","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/69//#testresult/ba22627ebbd8f334","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/auth/login (Caused by SSLError(SSLCertVerificationError(1, \"[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'api.agents.cardanoapi.io'. (_ssl.c:1000)\")))","time":{"start":1729679366899,"stop":1729679366899,"duration":0}},{"uid":"2c67cf059ff4b2d2","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/68//#testresult/2c67cf059ff4b2d2","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/auth/login (Caused by SSLError(SSLCertVerificationError(1, \"[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'api.agents.cardanoapi.io'. (_ssl.c:1000)\")))","time":{"start":1729672198605,"stop":1729672198605,"duration":0}},{"uid":"bc444e75b0c0b050","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/67//#testresult/bc444e75b0c0b050","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/auth/login (Caused by SSLError(SSLCertVerificationError(1, \"[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'api.agents.cardanoapi.io'. (_ssl.c:1000)\")))","time":{"start":1729667683697,"stop":1729667683697,"duration":0}},{"uid":"5cee5c08c3a99b42","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/66//#testresult/5cee5c08c3a99b42","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/auth/login (Caused by SSLError(SSLCertVerificationError(1, \"[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'api.agents.cardanoapi.io'. (_ssl.c:1000)\")))","time":{"start":1729664173095,"stop":1729664173095,"duration":0}},{"uid":"69ecf8806cef83c2","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/65//#testresult/69ecf8806cef83c2","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/auth/login (Caused by SSLError(SSLCertVerificationError(1, \"[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'api.agents.cardanoapi.io'. (_ssl.c:1000)\")))","time":{"start":1729574249392,"stop":1729574249392,"duration":0}},{"uid":"889aca47da17d069","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/64//#testresult/889aca47da17d069","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/auth/login (Caused by SSLError(SSLCertVerificationError(1, \"[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'api.agents.cardanoapi.io'. (_ssl.c:1000)\")))","time":{"start":1729508365704,"stop":1729508365704,"duration":0}},{"uid":"d504fc1235420507","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/63//#testresult/d504fc1235420507","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/auth/login (Caused by SSLError(SSLCertVerificationError(1, \"[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'api.agents.cardanoapi.io'. (_ssl.c:1000)\")))","time":{"start":1729506659955,"stop":1729506659955,"duration":0}},{"uid":"122009a8491c41c","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/62//#testresult/122009a8491c41c","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/auth/login (Caused by SSLError(SSLCertVerificationError(1, \"[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'api.agents.cardanoapi.io'. (_ssl.c:1000)\")))","time":{"start":1729245930148,"stop":1729245930148,"duration":0}},{"uid":"63bd765dbcec7c8e","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/61//#testresult/63bd765dbcec7c8e","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/auth/login (Caused by SSLError(SSLCertVerificationError(1, \"[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'api.agents.cardanoapi.io'. (_ssl.c:1000)\")))","time":{"start":1728386492781,"stop":1728386492781,"duration":0}},{"uid":"84aab5227cf5eed9","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/60//#testresult/84aab5227cf5eed9","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/auth/login (Caused by SSLError(SSLCertVerificationError(1, \"[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'api.agents.cardanoapi.io'. (_ssl.c:1000)\")))","time":{"start":1728380138774,"stop":1728380138774,"duration":0}},{"uid":"7b41d60e061f9540","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/59//#testresult/7b41d60e061f9540","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/auth/login (Caused by SSLError(SSLCertVerificationError(1, \"[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'api.agents.cardanoapi.io'. (_ssl.c:1000)\")))","time":{"start":1728370092942,"stop":1728370092942,"duration":0}}]},"tags":[]},"source":"88b756bf3b65ad31.json","parameterValues":[]}